modules/vaos/app/services/vaos/middleware/vaos_logging.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module VAOS
+  module Middleware
+    class VaosLogging < Faraday::Middleware
+      def initialize(app, service_name)
+        super(app)
+        @service_name = service_name
+      end
+
+      def call(env)
+        start_time = Time.current
+
+        @app.call(env).on_complete do |response_env|
+          jti = jti(env, response_env)
+          duration = Time.current - start_time
+          status = response_env.status
+
+          if status.between?(200..299) 
+            logger.info('vaos service call succeeded:',
+              duration: duration,
+              status: status,
+              jti: jti,
```

Comment:
jti includes ssn will this get filtered out?

Reply:
The JTI includes it or JWT includes it? I think it's just a unique identifier generated by VAMF. If we filter it, we can't reconcile logs -- i thought this was our unique identifier we go to VAMF with. It includes SSN really? 😬